src/Epub.ts

```diff
@@ -86,17 +87,21 @@ export default class Epub {
     const encryptionDoc = Epub.parseEncryptionDoc(encryptionStr);
 
     // ncx file
-    const relativeNcxPath = Epub.getNcxHref(opf);
-    const ncxPath = relativeNcxPath
-      ? fetcher.resolvePath(opfPath, relativeNcxPath)
-      : undefined;
+    const ncxPathRelativeToOpf = Epub.getNcxHref(opf);
+    // encrypted path
```

Comment:
Nit: here and line 121 I think these comments should be changed/removed since they're not encrypted paths, they're just the path relative to the root